### DISCUSSION TITLE
articles/giscus-is-the-key

### DISCUSSION THREAD

**Discussion by giscus** [tiagomichaelsousa/website#30] [Comments]
# articles/giscus-is-the-key

Introduction Disclaimer:  If took time to read this blog post and you have a GitHub account please react to this article 😊🙏 Whenever I have some time I like…

https://tiagomichaelsousa.dev/articles/giscus-is-the-key

**Comment by pourmand1376** [comment]
Thanks for the blog. I didn't know about this json file! Helped me a lot.

**Reply by tiagomichaelsousa** [reply]
I'm really happy that I was able to help you somehow 💪🥳

**Reply by secondsleep-blaine** [reply]
Same here! The `origins` restriction isn't something I would have known about. Thanks, Tiago!

**Reply by tiagomichaelsousa** [reply]
Welcome mate, glad to help ✌️
